text/0054-steering-committee-elections.md

```diff
@@ -0,0 +1,158 @@
+# Paketo Steering Committee Elections
+
+## Summary
+
+The project governance document will be amended to include a new process for
+electing the membership of the project Steering Committee.
+
+## Motivation
+
+The Paketo Steering Committee is the set of individuals who are responsible and
+accountable for guiding the Paketo project in a holistic manner towards
+achieving its goals. It currently consists of three members who help to
+administer the project and provide feedback and guidance on proposals that have
+project-wide impact.
+
+In the early years of the project, this group of individuals was composed of
+the project founders and then by engineers with significant experience with or
+contributions to the project. The technical process for voting in committee
+members is outlined in the [Paketo Governance
+RFC](https://github.com/paketo-buildpacks/rfcs/blob/main/text/0002-governance.md),
+but it is missing crucial detail on when and how these choices are made.
+
+As the project has evolved to support an ever growing number of users and
+contributors, it has become necessary to reassess this process and propose an
+alternative that will bring in a more diverse set of experiences and
+perspectives.
+
+## Detailed Explanation
+
+### Process Changes
+
+#### Membership
+
+The composition of the Steering Committee will align to the specifications,
+with some modifications, outlined in the [Cloud Foundry Foundation
+Charter](https://github.com/cloudfoundry/community/blob/main/governing-board/charter.md)
```

Comment:
Should we make a point to reference a specific commit of the CFF charter, rather than the main branch which could change over time? If we point to main, it seems like we'd essentially be accepting CFF governance changes automatically.